mesh/main.c

```diff
@@ -54,6 +54,29 @@ static const struct option main_options[] = {
 	{ }
 };
 
+char *CFG_DIR;
+int HCI_Idx = MGMT_INDEX_NONE;
+
+static void set_config_dir(const char *config_dir)
+{
+	CFG_DIR = (char *)config_dir;
+}
+
+static const char *get_config_dir(void)
+{
+	return CFG_DIR;
+}
+
+static void set_hci_idx(int index)
+{
+	HCI_Idx = index;
+}
+
+int get_hci_idx(void)
+{
+	return HCI_Idx;
+}
+
```

Comment:
These should be passed `user_data` instead of global variables.

Reply:
Done